homeassistant/components/atag/climate.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, coordinator, atag_id):
         self._attr_temperature_unit = coordinator.data.climate.temp_unit
 
     @property
-    def hvac_mode(self) -> str | None:
+    def hvac_mode(self) -> str | None:  # type: ignore[override]
```

Comment:
We should probably change the property in the parent climate class to allow returning `None` here to represent unknown state.